modin/experimental/core/execution/native/implementations/hdk_on_native/hdk_worker.py

```diff
@@ -163,9 +164,7 @@ def _hdk(cls) -> HDK:
         HDK
         """
         params = HdkLaunchParameters.get()
-        cls._preferred_device = (
-            "CPU" if bool(HdkLaunchParameters.get()["cpu_only"]) else "GPU"
-        )
+        cls._preferred_device = "CPU" if bool(params["cpu_only"]) else "GPU"
```

Comment:
no need to explicitly convert to `bool`
```suggestion
        cls._preferred_device = "CPU" if params["cpu_only"] else "GPU"
```

Reply:
@AndreyPavlenko please note